e2e/test/scenarios/dashboard-filters/dashboard-filters-sql-number.cy.spec.js

```diff
@@ -90,3 +90,69 @@ describe("scenarios > dashboard > filters > SQL > text/category", () => {
     });
   });
 });
+
+describe("scenarios > dashboard > filters > SQL > number", () => {
+  const questionDetails = {
+    name: "Question 1",
+    native: {
+      query:
+        "SELECT * from products where true [[ and price > {{price}}]] [[ and rating > {{rating}} ]] limit 5;",
+      "template-tags": {
+        price: {
+          type: "number",
+          name: "price",
+          id: "b22a5ce2-fe1d-44e3-8df4-f8951f7921bc",
+          "display-name": "Price",
+        },
+        rating: {
+          type: "number",
+          name: "rating",
+          id: "b22a5ce4-fe1d-44e3-8df4-f8951f7921bc",
```

Comment:
nit: I know ID isn't the focus here, but to avoid confusion can you use different IDs for the second parameter?

Reply:
it's different 😄

Reply:
Too hard to spot at a glance though. I think it's fairly simple to change this to a completely different UUID to avoid confusion (like I was).